### PR TITLE
Fix negative inputs, copied previous questions and zero quest setup

### PR DESCRIPTION
**Pull request by zoek1**
##### Description

1. Fix calculations when no quests exists (scenario: clean setup of gitcoin/web)
2. Adds min attribute to input number fields to allow only values greater than 0
New Quest Application _ Gitcoin
3. Provides clean inputs to new questions and answers
New Quest Application _ Gitcoin (1)
4. Limits the max number of questions to being created
DeepinScreenshot_20191026034454

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/5379

##### Testing

I added two minimal test to cover:
1. Create a quest requires a logged user
2. Negative seconds for questions are converted to positive

### CODE REVIEW

File: app/assets/v2/js/pages/quests.new.js [hunk 1]
```diff
@@ -1,25 +1,34 @@
 
 $(document).ready(function() {
+  var QUESTIONS_LIMIT=6;
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/quests.new.js]: `const QUESTIONS_LIMIT = 6;`

**Comment by owocki** [comment]
LGTM if the engineers are good with it

**Comment by zoek1** [comment]
I understand and apologize for the inconvenience. I promise to take care of that in the future.

**Comment by zoek1** [comment]
@danlipert @thelostone-mc i just added minor changes based on previous work on quest mobile view https://github.com/gitcoinco/web/pull/5422. I think this limits is more appropiate to display on mobile.

**Comment by zoek1** [comment]
just fixed conflicts, @octavioamu @danlipert if everything is ok, this could be merged?

File: app/quests/tests.py [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+            'question[]': ['one?']})
+        response = self.client.post(reverse('newquest'),
+                                    data,
+                                    content_type="application/x-www-form-urlencoded",
+                                    HTTP_USER_AGENT='none')
+
+        self.assertTrue(status.is_success(response.status_code))
+
+    def test_new_quests_should_redirect_to_login_when_no_user_is_logged(self):
+        """Test abs function on seconds to prevent set negative second to respond on quests questions"""
```
Comment (danlipert) [inline app/quests/tests.py]: I think this description needs to be updated

Comment (zoek1) [reply app/quests/tests.py]: @danlipert done!

**Pull request merged**